libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -169,8 +169,8 @@ public Bundle getTranslations() {
             }
 
             String fieldName = stringField.getName();
-            // Filter out all strings that are not prefixed with `gutenberg_mobile_`
-            if (!fieldName.startsWith("gutenberg_mobile_")) {
+            // Filter out all strings that are not prefixed with `gutenberg_native_`
+            if (!fieldName.startsWith("gutenberg_native_")) {
```

Comment:
Given this change we actually need to update `values/strings.xml` in this PR if we don't want to lose the gutenberg translations in the next release.

Reply:
Let's target https://github.com/wordpress-mobile/WordPress-Android/pull/10769 instead of `develop` se we can have access to the gutenberg-mobile strings.xml  and thus re-generate the main strings.xml